src/com/lilithsthrone/game/Game.java

```diff
@@ -1002,6 +1002,10 @@ public static void importGame(File file) {
 										className = className.substring(0, lastIndex) + ".misc" + className.substring(lastIndex, className.length());
 									}
 								}
+								if(Main.isVersionOlderThan(loadingVersion, "0.3.21")) {
```

Comment:
@Innoxia: Feel free to change that to whatever the next version is when adding that PR.

Reply:
Will do!